packages/dds/sequence/README.md

```diff
@@ -148,6 +158,26 @@ specified range. Setting a property key to null will remove that property from t
 Whenever an operation is performed on a sequence a *sequenceDelta* event will be raised. This event provides the ranges
 affected by the operation, the type of the operation, and the properties that were changes by the operation.
 
+```typescript
+sharedString.on("sequenceDelta", ({ deltaOperation, ranges, isLocal }) => {
+    if (isLocal) {
+        // ex: app code may have already synced local operations to their model at the time they were applied
```

Comment:
we've seen good success where local deltas are consumed the same way as remote deltas. so basically on key press just modify the sequence directly, then let your event handlers update you view. maybe too much details for this document though

Reply:
yeah, that's closer to the pattern I'd expect. Might as well make the example realistic, I'll replace this with something about adding the delta to the undo stack in the `isLocal` block and then call some updateView code regardless.